csv/src/test/java/org/apache/metamodel/csv/CsvDataContextTest.java

```diff
@@ -833,8 +831,14 @@ public void testCustomColumnNames() throws Exception {
         final String thirdColumnName = "third";
         final String fourthColumnName = "fourth";
 
+        final ColumnType firstColumnType = ColumnType.STRING;
```

Comment:
I don't think it adds real value to put these constants into the xxxColumnType variables, especially since they're only used once. It doesn't make the code more understandable or more elegant.